tests/cucumber/browser/test.js

```diff
@@ -1,9 +1,8 @@
 const assert = require('assert');
-const sha256 = require('js-sha256');
```

Comment:
was this a bug in the test implementation that you are fixing?

Reply:
It was an import used by the template tests so I just removed it in this PR